packages/avatars/src/styled/StyledStatusIndicator.ts

```diff
@@ -17,7 +17,7 @@ export interface IStatusIndicatorProps extends Omit<IAvatarProps, 'badge' | 'isS
   borderColor?: string;
 }
 
-const COMPONENT_ID = 'avatars.status_indicator';
+const COMPONENT_ID = 'avatars.avatar.status-indicator';
```

Comment:
[nit] we try to avoid component ID renames, when possible, as it can mess with existing [targeting](https://garden.zendesk.com/components/theme-provider#targeting) customizations.